src/server/storage/storageAPI.js

```diff
@@ -126,22 +122,9 @@ const setup = (app: Router, storage: StorageAPI) => {
     '/user/delete',
     passport.authenticate('jwt', { session: false }),
     wrapAsync(async (req, res, next) => {
-      const { user, log, body } = req
-      const { zoomSignature, zoomId } = body
+      const { user, log } = req
       log.info('delete user', { user })
 
-      if (zoomId && zoomSignature) {
```

Comment:
why was this removed?
in client code we still need to use our special zoomId as id when enrolling the user